### PR TITLE
Update ja.po

### PR DESCRIPTION
**Pull request by myyourcaramel**

### CODE REVIEW

File: translations/ja.po [hunk 1]
```diff
@@ -56,16 +56,16 @@ msgstr "**ペンデュラムスケール**: ${ formattedScale }"
 #: src\card.ts:240
 #, javascript-format
 msgid "Password: ${ card.password } | Konami ID #${ card.konami_id }"
-msgstr ""
+msgstr "パスワード: ${ card.password } | Konami ID #${ card.konami_id }"
 
 #: src\card.ts:242
 msgid "No password | Konami ID #${ card.konami_id }"
-msgstr ""
+msgstr "パスワード: 未設定 | Konami ID #${ card.konami_id }"
```
Comment (kevinlul) [inline translations/ja.po:63]: Are you sure you want to go with "パスワード: 未設定" for "No password"? There are certain cards that have never had a password printed on them, namely the Egyptian Gods and ["マジマジ☆マジシャンギャル"](https://yugipedia.com/wiki/Magi_Magi_%E2%98%86_Magician_Gal)

File: translations/ja.po [hunk 2]
```diff
@@ -56,16 +56,16 @@ msgstr "**ペンデュラムスケール**: ${ formattedScale }"
...
 
 #: src\card.ts:244
 #, javascript-format
 msgid "Password: ${ card.password } | Not yet released"
-msgstr ""
+msgstr "パスワード: ${ card.password } | Konami ID: 未設定"
```
Comment (kevinlul) [inline translations/ja.po:68]: Are you sure you would like to go with `Konami ID: 未設定` for "Not yet released"? It is labelled "未発売" below.

File: translations/ja.po (hunk 1 above)
Comment (myyourcaramel) [reply translations/ja.po:63]: I wasn't aware that "No password" here is meant for these cards XD

Comment (myyourcaramel) [reply translations/ja.po:63]: I'll fix this

File: translations/ja.po (hunk 2 above)
Comment (myyourcaramel) [reply translations/ja.po:68]: Wow good catch XD

Actaully I wasn't sure how(and when) a Konami ID is set. If a card with password without Konami ID means the card is not yet released, "Not yet released" should be used. But I wasn't sure if it's always so.

But apparently it's almost always the case, so I should use "未発売(Not yet released)" here.

**Review by kevinlul — APPROVED** [review]
Whoops, forgot about this!

**Pull request merged**